docs/mkdocs/docs/developers/paig-client/code-examples/milvus-integration/a-sample-tutorial.md

```diff
@@ -0,0 +1,253 @@
+---
+title: Sample Tutorial
+---
+
+# Milvus Integration Sample Tutorial
+
+This tutorial shows you how to prevent unauthorized access to your data by:
+
+- Restricting access based on users and groups.
+- Restricting access based on metadata policies.
+
+## Prerequisite tasks
+
+### Install Docker 
+Make sure you have Docker installed on your system. You can download it from the [Docker website](https://www.docker.com/products/docker-desktop/).
+
+### Set up the tutorial directory
+Download below files and place them in new directory where you want to run the tutorial.
+
+ - <a href="files/docker-compose.yml" download="docker-compose.yml">docker-compose.yml</a>
+ - <a href="files/requirements.txt" download="requirements.txt">requirements.txt</a>
+ - <a href="files/create_milvus_collection.py" download="create_milvus_collection.py">create_milvus_collection.py</a>
+ - <a href="files/query_milvus_collection.py" download="query_milvus_collection.py">query_milvus_collection.py</a>
+
+### Start Milvus and Attu server
+!!! note "Note"
+    If you have your own Milvus server running and you want to use that, you can skip this step.
+
+Run the following command in the directory where you placed the files.
+    ```bash
+    docker-compose up -d
+    ```
+
+### Install the required Python packages
+Run the following command to install the required Python packages. 
+
+For this tutorial, we will be installing the `milvus`, `openai` and `paig_client` packages.
+
+- Create a virtual environment and activate it.
+  ```bash
+  python3 -m venv venv
+  source venv/bin/activate
+  ```
+- Install the required packages.
+  ```bash
+  pip install -r requirements.txt
+  ```
+
+### Export your OpenAPI key
+Export your OpenAI API key as an environment variable to use ir further in the code to create the embeddings.
+   ```bash
+   export OPENAI_API_KEY=<your_openapi_key>
+   ```
+
+### Create Milvus collection
+Run the following command to create a Milvus collection with sample data.
+!!! note "Note"
+    If you are using your own Milvus server, make sure to update the `MILVUS_HOST` and `MILVUS_PORT` in the `create_milvus_collection.py` script.
```

Comment:
instead of changing file, better to read env vars and instruct to setup env vars

Reply:
updated code to read from env variables